libs/platform/src/lib/components/input-group/input-group.component.ts

```diff
@@ -0,0 +1,154 @@
+import {
+    Component,
+    Input,
+    ChangeDetectionStrategy,
+    ContentChildren,
+    QueryList,
+    ElementRef,
+    ViewEncapsulation,
+    ChangeDetectorRef,
+    Renderer2,
+    AfterViewInit,
+    OnInit,
+    AfterContentInit
+} from '@angular/core';
+import { startWith } from 'rxjs/operators';
+
+import { BaseComponent } from '../base';
+import { InputComponent } from '../form/input/input.component';
+import { ContentDensity, Status } from '../form/form-control';
+
+import { InputGroupConfig } from './input-group.config';
+import { InputGroupAddonComponent } from './addon/addon.component';
+
+const CSS_CLASS_NAME = {
+    inputGroup: 'fdp-input-group',
+    inputGroupInnerInput: 'fd-input-group__input'
+} as const;
```

Comment:
Does this bring any value `as const;`?

Reply:
It makes `CSS_CLASS_NAME` props `readonly` as result it's not allowed to change its values.
Also it's convenient to hover property and see what the value actually is:
![image](https://user-images.githubusercontent.com/3263744/92303856-abc73400-ef81-11ea-9c84-625b88bec574.png)